lib/cloud_five_push/version.rb

```diff
@@ -1,3 +1,3 @@
 module CloudFivePush
-  VERSION = "0.1.1"
+  VERSION = "1.0.0"
```

Comment:
Not sure about this, but it is technically a breaking change, so I figured we bump the major version number.